### PR TITLE
[RHACS] ROX-13330 add QRadar and ServiceNow integrations

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):

- `rhacs-docs`
- `rhacs-docs-3.73`
- `rhacs-docs-3.72`
- `rhacs-docs-3.71`
- `rhacs-docs-3.74`

[Issue](https://issues.redhat.com/browse/ROX-13330)

Link to docs preview:

- [Integrating with QRadar](https://openshift-docs-oci65apfm-kcarmichael08.vercel.app/openshift-acs/master/integration/integrate-with-qradar.html)
- [Integrating with ServiceNow](https://openshift-docs-oci65apfm-kcarmichael08.vercel.app/openshift-acs/master/integration/integrate-with-servicenow.html)

QE review:
- [ ] QE has approved this change. (**N/A** - ACS has no QE, but SME approved via Slack.)

**Note** External links approved by Support via Slack.

### CODE REVIEW

File: integration/integrate-with-qradar.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="integrate-with-qradar"]
+= Integrating with QRadar
+include::modules/common-attributes.adoc[]
+:context: integrate-with-qradar
+
+toc::[]
+
+[role="_abstract"]
+You can configure {product-title-short} to send events to QRadar by configuring a generic webhook integration in {product-title-short}.
```
Comment (jboxman-rh) [inline integration/integrate-with-qradar.adoc:10]: Do we want to expand RHACS for the first usage?

File: integration/integrate-with-qradar.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+The following steps represent a high-level workflow for integrating {product-title-short} with QRadar:
+
+. In {product-title-short}:
+.. Configure the generic webhook. 
++
+[NOTE]
+====
+When configuring the integration in {product-title-short}, in the *Endpoint* field, use the following example as a guide: `<URL to QRadar Box:<Port of Integration>`.
```
Comment (jboxman-rh) [inline integration/integrate-with-qradar.adoc:19]:
```suggestion
When configuring the integration in {product-title-short}, in the *Endpoint* field, use the following example as a guide: `<URL to QRadar Box>:<Port of Integration>`.
```

Is there a missing angle bracket?

File: integration/integrate-with-qradar.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+====
+.. Identify policies for which you want to send notifications, and update the notification settings for those policies. 
+. If QRadar does not automatically detect the log source, add an {product-title-short} log source on the QRadar Console. See link:https://www.ibm.com/docs/en/dsm?topic=configuration-red-hat-advanced-cluster-security-kubernetes[Red Hat Advanced Cluster Security for Kubernetes] for more information.
+
+
```
Comment (jboxman-rh) [inline integration/integrate-with-qradar.adoc:24]: Extra newlines?

File: integration/integrate-with-servicenow.adoc [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+:context: integrate-with-servicenow
+
+toc::[]
+
+[role="_abstract"]
+You can configure {product-title-short} to send events to ServiceNow by configuring a generic webhook integration in {product-title-short}.
+
+The following steps represent a high-level workflow for integrating {product-title-short} with ServiceNow:
+
+. In ServiceNow, configure a REST API endpoint to use in {product-title-short}. See link:https://www.redhat.com/en/blog/how-integrate-red-hat-advanced-cluster-security-kubernetes-servicenow[How to integrate Red Hat Advanced Cluster Security for Kubernetes with ServiceNow] for more information.
```
Comment (jboxman-rh) [inline integration/integrate-with-servicenow.adoc:14]: I think we're supposed to confirm with support that this link is okay.

**Review by jboxman-rh — COMMENTED** [review]
@kcarmichael08, I've left some comments. Thanks!

File: integration/integrate-with-qradar.adoc (hunk 2 above)
Comment (kcarmichael08) [reply integration/integrate-with-qradar.adoc:19]: That's exactly how they have it in the doc, so I don't know. I can check.

Comment (kcarmichael08) [reply integration/integrate-with-qradar.adoc:19]: Confirmed from our ACS team that it needs to be URL:port (or an endpoint) so yes, good catch, needs another bracket!

File: integration/integrate-with-servicenow.adoc (hunk 4 above)
Comment (kcarmichael08) [reply integration/integrate-with-servicenow.adoc]: Confirmed with Support that this link and the other link are OK.

**Pull request merged**